scripts/sample-data.js

```diff
@@ -0,0 +1,145 @@
+const chalk = require('chalk')
+const urllib = require('url')
+const fs = require('fs')
+const path = require('path')
+const Web3 = require('web3')
+const Origin = require('../dist/index.js')
+
+
+// This script creates randomly generated sample listings for testing purposes.
+// Currently only written to run against local setup.
+
+// 1) Create a couple of user accounts
+// 2) Create a bunch of listings
+// 3) Users make offers on other's listings
+
+// Local Usage:
+// scripts/sampleData.js <number of listings> <number of offers>
+// docker exec -ti origin-js node scripts/sampleData.js 42 17
+
+const NUMBER_OF_LISTINGS = parseInt(process.argv[2] || '30')
+const NUMBER_OF_OFFERS = parseInt(process.argv[3] || NUMBER_OF_LISTINGS / 2)
+
+const NAMES = [
+  'Stan',
+  'Micah',
+  'Tyler',
+  'Josh',
+  'Daniel',
+  'Matthew',
+  'Kay',
+  'Пётр',
+  '계룡/鷄龍',
+  'Urist'
+]
+const CATEGORIES = ['Housing', 'Collectibles', 'For Sale', 'Services']
+const wordsFilePath = path.join(__dirname, "sample-words.json")
+const WORDLIST = JSON.parse(fs.readFileSync(wordsFilePath))
+
+function randomWord() {
+  return WORDLIST[Math.floor(Math.random() * WORDLIST.length)]
+}
+
+function randomTitleWord() {
+  const word = randomWord()
+  return word.charAt(0).toUpperCase() + word.slice(1)
+}
+
+/*
+ * Setup
+ */
+
+const web3Provider = new Web3.providers.HttpProvider('http://localhost:8545')
+const web3 = new Web3(web3Provider)
+const ipfsUrl = urllib.parse('http://origin-js:8080')
```

Comment:
super nit: maybe add a comment explaining that host "origin-js" is the virtual host created by the origin-js container when it runs in origin-box.